IntegrationTests/UITests/Script/ScriptRunnerTests.cs

```diff
@@ -187,13 +181,17 @@ public void RunScript_with_arguments_with_s_option_with_RevisionGrid_with_select
             RunFormTest(formBrowse =>
             {
                 // wait until the revisions are loaded
-                while (formBrowse.RevisionGridControl.LatestSelectedRevision == null)
+                while (formBrowse.RevisionGridControl.GetSelectedRevisions().Count == 0)
                 {
                     Application.DoEvents();
```

Comment:
This is not the correct way to wait for background operations to complete. It should be using `JoinPendingOperationsAsync`.